src/main/resources/test_files/runtime-spark/RumbleML/EstimatorTests/EstimatorTODO

```diff
@@ -0,0 +1,40 @@
+DONE: estimatorFullClassNames.put("GeneralizedLinearRegression", "org.apache.spark.ml.regression.GeneralizedLinearRegression");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("IDF", "org.apache.spark.ml.feature.IDF");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("Imputer", "org.apache.spark.ml.feature.Imputer");
+DONE: estimatorFullClassNames.put("IsotonicRegression", "org.apache.spark.ml.regression.IsotonicRegression");
+DONE: estimatorFullClassNames.put("KMeans", "org.apache.spark.ml.clustering.KMeans");
+DONE: estimatorFullClassNames.put("LDA", "org.apache.spark.ml.clustering.LDA");
+DONE: estimatorFullClassNames.put("LinearRegression", "org.apache.spark.ml.regression.LinearRegression");
+DONE: estimatorFullClassNames.put("LinearSVC", "org.apache.spark.ml.classification.LinearSVC");
+DONE: estimatorFullClassNames.put("LogisticRegression", "org.apache.spark.ml.classification.LogisticRegression");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("MaxAbsScaler", "org.apache.spark.ml.feature.MaxAbsScaler");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("MinHashLSH", "org.apache.spark.ml.feature.MinHashLSH");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("MinMaxScaler", "org.apache.spark.ml.feature.MinMaxScaler");
+DONE: estimatorFullClassNames.put("MultilayerPerceptronClassifier", "org.apache.spark.ml.classification.MultilayerPerceptronClassifier");
+DONE: estimatorFullClassNames.put("NaiveBayes", "org.apache.spark.ml.classification.NaiveBayes");
+DONE: estimatorFullClassNames.put("OneHotEncoderEstimator", "org.apache.spark.ml.feature.OneHotEncoderEstimator");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("OneVsRest", "org.apache.spark.ml.classification.OneVsRest");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("PCA", "org.apache.spark.ml.feature.PCA");
+DONE: ACTION UNCLEAR: estimatorFullClassNames.put("Pipeline", "org.apache.spark.ml.Pipeline");
+DONE: estimatorFullClassNames.put("QuantileDiscretizer", "org.apache.spark.ml.feature.QuantileDiscretizer");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("RFormula", "org.apache.spark.ml.feature.RFormula");
+DONE: estimatorFullClassNames.put("RandomForestClassifier", "org.apache.spark.ml.classification.RandomForestClassifier");
+DONE: estimatorFullClassNames.put("RandomForestRegressor", "org.apache.spark.ml.regression.RandomForestRegressor");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("StandardScaler", "org.apache.spark.ml.feature.StandardScaler");
+DONE: estimatorFullClassNames.put("StringIndexer", "org.apache.spark.ml.feature.StringIndexer");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("TrainValidationSplit", "org.apache.spark.ml.tuning.TrainValidationSplit");
+DONE: MISSING IMPLEMENTATION: estimatorFullClassNames.put("VectorIndexer", "org.apache.spark.ml.feature.VectorIndexer");
+DONE: estimatorFullClassNames.put("Word2Vec", "org.apache.spark.ml.feature.Word2Vec");
```

Comment:
@ghislainfourny Overview of the status of estimators

Reply:
Wonderful!